frameworks/kafka/docs/upgrade.md

```diff
@@ -65,9 +65,9 @@ First, make a note of the existing protocol and log versions. Upgrade your servi
         }
     }
 
-Now, upgrade to the package `2.0.0-0.11.0.0` that provides the new code, Kafka 0.11.0.0. 
+Now, upgrade to the package `1.1.27-0.11.0-beta` that provides the new code, Kafka 0.11.0.0. 
```

Comment:
Now, upgrade to package `1.1.27-0.11.0-beta`, which provides the new code, Kafka 0.11.0.0.